scio-redis/src/main/scala/com/spotify/scio/redis/RedisIO.scala

```diff
@@ -41,17 +49,12 @@ object RedisConnectionOptions {
   private[redis] def toConnectionConfig(
     connectionOptions: RedisConnectionOptions
   ): RedisConnectionConfiguration = {
-    var config = RedisConnectionConfiguration
+    val config = RedisConnectionConfiguration
       .create(connectionOptions.host, connectionOptions.port)
       .withTimeout(connectionOptions.timeout.getMillis.toInt)
+      .withSSL(StaticValueProvider.of(connectionOptions.useSSL))
 
-    if (connectionOptions.useSSL) {
-      config = config.enableSSL()
-    }
-
-    connectionOptions.auth.foreach(a => config = config.withAuth(a))
-
-    config
+    connectionOptions.auth.fold(config)(config.withAuth(_))
```

Comment:
I guess `(_)` could be dropped here?

Reply:
it's ok not to do eta expansion as it will be removed in the future

Reply:
😢